src/core/Table/Table.tsx

```diff
@@ -373,118 +431,162 @@ export const Table = <
     [isRowDisabled, isSelectable, selectRowOnClick, dispatch, onRowClick],
   );
 
+  React.useEffect(() => {
+    setPageSize(pageSize);
+  }, [pageSize, setPageSize]);
+
+  const paginatorRendererProps: TablePaginatorRendererProps = React.useMemo(
+    () => ({
+      currentPage: state.pageIndex,
+      pageSize: state.pageSize,
+      totalRowsCount: rows.length,
+      density,
+      isLoading,
+      onPageChange: (page) => {
+        gotoPage(page);
+        tableRef.current?.scrollIntoView({
+          block: 'start',
+          inline: 'nearest',
+          behavior: 'smooth',
+        });
```

Comment:
Table should already be in view, no?

Reply:
It is but if you check the story yourself, it will scroll to the top of the table. Not sure if it is ok or not but personally I don't like when I switch the page and have to scroll to the top myself 😀

Reply:
It doesn't work when using keyboard. Or is that intentional?
![paginator](https://user-images.githubusercontent.com/9084735/136392868-9e647999-682a-45b9-8d28-c7609eec3116.gif)